Subject: [PATCH ltp-full-20080831] don't use libposix1b
From: Robert Schwebel <r.schwebel@pengutronix.de>

The semaphores tests of the open_posix_testsuite want to link again
libposix1b, but I don't have any idea why it tries to do so. However, it
results in:

gcc -Wall -O2 -g -I -L/usr/lib -I../../include multi_con_pro.c -o multi_con_pro.test -lposix1b -lpthread
/usr/bin/ld: cannot find -lposix1b

This patch removes the bogus library.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 testcases/open_posix_testsuite/stress/semaphores/Makefile |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: ltp-full-20080831/testcases/open_posix_testsuite/stress/semaphores/Makefile
===================================================================
--- ltp-full-20080831.orig/testcases/open_posix_testsuite/stress/semaphores/Makefile
+++ ltp-full-20080831/testcases/open_posix_testsuite/stress/semaphores/Makefile
@@ -1,12 +1,11 @@
 INCLUDE = -I../../include
-LIB=-lposix1b
 
 CFLAGS=-Wall -O2 -g
 
 all:  multi_con_pro.test
 
 %.test : %.c
-	$(CC) $(CFLAGS) $(INCLUDE) $< -o $@ $(LIB) -lpthread  
+	$(CC) $(CFLAGS) $(INCLUDE) $< -o $@ -lpthread
 clean: 
 	rm *.test
 
